packages/react/src/patterns/sub-patterns/CardGroup/__stories__/CardGroup.stories.js

```diff
@@ -24,7 +24,7 @@ storiesOf('Patterns (Sub-Patterns)|CardGroup', module)
   .add('default', () => {
     const cardTypes = Object.keys(cards);
 
-    const type = select('cardType', cardTypes, cardTypes[0]);
+    const type = select('Card type', cardTypes, cardTypes[0]);
```

Comment:
@kennylam one of the things I started doing is add in parenthesis what the actual prop name is in the storybook knobs. Carbon team seems to do that, and I already refactored the DotcomShell/Masthead/Footer to follow this.

Reply:
sounds good i'll change